_announcements/2020-09-18

```diff
@@ -0,0 +1,20 @@
+---
+layout: default
+title: Air Quality Update
+date: 2019-09-18 09:15
+author: Klahanie
+comments: true
+categories: [announcements]
+show-title: true
+---
+The Lakeside pool will be today and will be resuming to the normal schedule with lap swim offered in the morning and evenings as well as open instructional swim. The air quality has improved and is between the ranges of 100-150 as seen on the purple air website using this link: https://www.purpleair.com. Attached is also our September Pool Schedule.
+
+If you have any questions feel free to contact our pool managers at swimlessons@klahanie.com or call the Lakeside Pool at 425-391-8503. 
+This is the article body. 
+
+* [Link](https://link.com)
+* [Link](https://link.com)
+
+### Subheader
+
+Another paragraph
```

Comment:
```suggestion
```

@SharonManyara Looks like you left the template in here, do you mind deleting? You can accept my suggestion to do so.

Reply:
Oops, just a second